app/controllers/reviews_controller.rb

```diff
@@ -46,8 +48,12 @@ def set_review
       @review = Review.find(params[:id])
     end
 
+    def set_medium
+      @medium = Medium.find(params[:medium_id])
+    end
```

Comment:
Might want to handle a nil @medium here just so you're not gathering pain in the controller actions